### ISSUE TITLE
1.15.2 - Dungeons not generating on server

### ISSUE THREAD

**Issue by Partonetrain** [xyroc/DungeonCrawl#15]
DungeonCrawl-1.15.2-1.6.1
Forge 31.1.1

To reproduce:
Create world with set seed in singleplayer. Use /locate to find dungeon.
Teleport to the same place in the server, there won't be a dungeon.
If you do /locate in the dedicated server, server freezes, but doesn't crash (??), as it can't find a dungeon.

**Comment by Partonetrain** [comment]
Deleted world and regenerated, it's there. I think I generated the world at first without the mod. Sorry!

**Comment by xyroc** [comment]
> If you do /locate in the dedicated server, server freezes, but doesn't crash (??), as it can't find a dungeon.

Did it just not find a dungeon or was there an error?

**Comment by Partonetrain** [comment]
The command didn't return anything, not even "can't find structure". No error, or crash, the server just froze up. Didn't even crash from the server watchdog. No idea...

I think this is a technicality with the /locate command, not DungeonCrawl, so I closed the issue. The same thing happened when I used /locate for Mine and Slash structures in the overworld - where they don't generate. But I don't know why it happened with DungeonCrawl.

It hasn't happened again since I regenerated the world.
